lib/contentful/meta_data.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Contentful.MetaData do
```

Comment:
I was struggling with this one, [since `:sys` exists](https://erlang.org/doc/man/sys.html) and this  would end with a double meaning in the Elixir environment.

`Contentful.System` could work, maybe?

Reply:
Maybe. Or what about `SysData` (just so the name is closer to the property somehow)?